src/defines.asm

```diff
@@ -405,6 +405,7 @@
 !COMPRESSED_GRAPHICS = #$0001
 !COMPRESSED_PALETTES = #$0002
 !COMPRESSED_PALETTES_8BIT = #$02
+!COMPRESSED_TABLES = #$0004
```

Comment:
Freed up enough space that we can use raw tile tables too.  Just about all of PJBoy's uncompressed data is being used for preset loading.  The tile tables don't make that much difference though; if we get low on space we could remove this.